app/views/shared/_user.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="row align-items-center">
+
+        <div class="col-4">
+                <img src=<%=user.photo%> class="user-pic" alt=<%=user.name%> >
+        </div>
+
+        <div class="card col-8 d-flex flex-column justify-content-between">
+
+
+                <div class="col-7 d-flex justify-content-center flex-column">
+                        <h2 class = "h1">
+                                <%=user.name%>
+                        </h2>
+                </div>
+
+                <div class="col-5 align-self-end d-flex">
+                        <p>Number of posts:
+                                <%=user.posts_counter%>    
+                        </p>
+                </div>
+
+        <div>
+</div>
```

Comment:
Please be more specific

Reply:
The code is hard to read because of the spacing between elements and its child/children. Try using a tab for an element and for its child so that the code have a good line spacing. Hope this helps.